tendrl-api.spec

```diff
@@ -1,4 +1,11 @@
-Name: tendrl-api
+%global name tendrl-api
+%global app_group %{name}
+%global app_user %{name}
+%global install_dir %{_datadir}/%{name}
+%global config_dir %{_sysconfdir}/tendrl
```

Comment:
@anivargi  Isnt the config dir /etc/tendrl/api  instead of /etc/tendrl ?

Reply:
The config dir is correct and it was the same before, the api looks at /etc/tendrl/etc.yml